src/NotificationManager/index.js

```diff
@@ -0,0 +1,133 @@
+import React, {
+   createContext,
+   useContext,
+   useEffect,
+   useRef,
+   useState
+} from 'react'
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import PropTypes from 'prop-types'
+import { createPortal } from 'react-dom'
+import { Transition, TransitionGroup } from 'react-transition-group'
+import OIOContext from '../OIOProvider/context'
+import { NotificationInline, View } from '..'
+
+const ANIMATION_DURATION = 600
+
+// =================================================
+// Context
+// =================================================
+
+const NotificationManagerContext = createContext()
+
+// =================================================
+// Provider
+// =================================================
+
+const NotificationManagerProvider = ({ children }) => {
+   const oioContext = useContext(OIOContext)
+   const notificationElem = useRef(document.createElement('div'))
```

Comment:
Could be named better, like `notificationContainerDiv` or something? Right now it sounds more like this is the notification itself as opposed to the container

(Yes I realize this was originally in my prototype PR :))